atom/common/options_switches.cc

```diff
@@ -182,7 +182,7 @@ const char kOpenerID[]         = "opener-id";
 const char kScrollBounce[]     = "scroll-bounce";
 const char kHiddenPage[]       = "hidden-page";
 const char kNativeWindowOpen[] = "native-window-open";
-const char kWebviewTag[]       = "webview-tag";
+const char kWebviewElement[]   = "webview-element";
```

Comment:
I am not sure if I can rename this, I haven't found the place parsing the command line option yet.